Makefile

```diff
@@ -4,7 +4,7 @@ export
 
 CWD := $(shell pwd)
 
-MAKE := make -f Build/Makefile -j 8
+MAKE := make -f Build/Makefile -j $(shell nproc)
```

Comment:
`nproc ` is not available on OSX by default so this will fail on OSX. `sysctl -n hw.physicalcpu` is euqivalent of `nproc` for OSX as per Google. Which means we would have to call a different command based on the OS. Not sure if it is worth it.

@pgravel-com do you have an opinion here?

Reply:
I don’t remember installing anything for `nproc`, my bad. 

I used it here because I am so used to seeing it everywhere in Makefiles.

Reply:
I updated the branch with a fix proposition. I check if the host is Darwin or Linux and use the corresponding executable to get `nproc`